ext/travisci/install_puppetserver.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
```

Comment:
Sounds good (and if you're not famiiar with it, note that pipefail can behave in ways you might not expect wrt, for example, grep, i.e. cause the rest of the pipeline to fail when a negative result was not an error).